EZEE-1901: Implement Fastly Shielding (#1848)
ezsystems/developer-documentation: docs/guide/cache/symfony_reverse_proxy.md

    Use of Varnish or Fastly is a requirement for a [Clustering](../clustering.md) setup, as Symfony Proxy does not support sharing cache between several application servers.

## VCL base files

For reverse proxies to work properly with your installation, you need to install the corresponding VCL files for your
HTTP Cache.

- [Varnish VCL xkey example](https://github.com/ezsystems/ezplatform-http-cache/blob/2.0/docs/varnish/vcl)
- Fastly VCLs can be found in `vendor/ezsystems/ezplatform-http-cache-fastly/fastly`

>   - `ez_main.vcl` needs to installed as the main custom vcl

Reviewer: ```suggestion
  - `ez_main.vcl` needs to installed as the main custom VCL
```

>   - `ez_user_hash.vcl` also needs to be installed as custom vcl

Reviewer: ```suggestion
  - `ez_user_hash.vcl` also needs to be installed as custom VCL
```

  - `snippet_re_enable_shielding.vcl` needs to be installed as snippet. See comments in the .vcl file itself for
    instructions on how to install it.

>   - As of Ibexa DXP v3.3.24 and V4.1.6 [Fastly Shielding](https://docs.fastly.com/en/guides/shielding) is supported. The
>     provided .vcl files will work with both shielding enabled or not.

Reviewer: ```suggestion

The provided .vcl files will work with both with [Fastly Shielding](https://docs.fastly.com/en/guides/shielding) enabled and without it.
```

Author: I agree that your sentence is better. However, may we just add one line when actually shielding was added ( as this was done in a subrelease of both 3.3.x and 4.1 ?
```suggestion

The provided .vcl files will work both with [Fastly Shielding](https://docs.fastly.com/en/guides/shielding) enabled and without it.

!!! tip
    Support for Fastly Shielding was added in Ibexa DXP v3.3.24 and V4.1.6
```

If you don't want it, I am fine with your suggestion.
The tip I suggested above will be followed by another tip, so that is maybe not so good either?

Reviewer: Sounds good, no issues with two tips in a row.
